code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -864,6 +864,19 @@ generate/load female uniform sprites matching all previously decided variables
 					"params" = displacement_map_filter(cut_legs_mask, x = 0, y = 0, size = 4),
 				),
 			))
+		if(MONKEY_HEIGHT_TALL)
```

Comment:
`MONKEY_HEIGHT_TALL` is the same height as `HUMAN_HEIGHT_DWARF` (`6`) which means the latter of these two switch cases will never be entered and is now dead code, you will need to fix that.